actioncable/test/client_test.rb

```diff
@@ -8,6 +8,13 @@
 
 require "active_support/hash_with_indifferent_access"
 
+# test with iodine if platform allows
+begin
+  require "iodine"
+rescue LoadError
+  nil
```

Comment:
Can omit this line. An empty rescue is common to see.

Reply:
Sure 👍🏻